release-1.8/release_notes_draft.md

```diff
@@ -44,3 +44,37 @@ please check out the [release notes guidance][] issue.
 ## **Deprecations**
 
 ## **Notable Features**
+
+### **Node Components**
+#### kubelet
```

Comment:
Hmm, problem is that phrase doesn't mean anything without context.

I was trying to be broad enough that future numa and device aware placement could go in this same doc page.  Also tried to disambiguate "placement" with "workload placement" rather than "pod placement" which _would_ be a kube scheduler thing.

Not sure what would be better...